src/components/e-input.vue

```diff
@@ -216,8 +234,8 @@
 
       window.addEventListener('resizeend', this.setSlotSpacings);
 
-      if (this.uncontrolled && this.$refs.input) {
-        this.$refs.input.value = this.standalone ? this.internalValue : this.value;
+      if (this.uncontrolled && this.input) {
+        this.input.value = this.standalone ? this.internalValue.toString() : this.value.toString();
```

Comment:
This should not be needed any more, since you the defined the prop as type 'string`.